samples/transient-clients/Core_6/Client/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.SignalR.Client;
+
+static class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.TransientClients.Client";
+
+        var url = "http://localhost:8080";
+
+        var hubConnection = new HubConnection(url);
+        var stockHubProxy = hubConnection.CreateHubProxy("StockTicksHub");
+        stockHubProxy.On<StockTick>("StockTick", stock => Console.WriteLine($"Stock update for {stock.Symbol} at {stock.Timestamp:O}. Press any key to exit."));
```

Comment:
Yes, if you run the sample the console quickly fills with these messages. I thought adding this would be helpful, but I don't mind removing it if others feel otherwise.